trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/model/ServerInfoResponse.java

```diff
@@ -0,0 +1,77 @@
+/**********************************************************************
+ * Copyright (c) 2024 Ericsson
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License 2.0 which
+ * accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+package org.eclipse.tracecompass.incubator.internal.trace.server.jersey.rest.core.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+/**
```

Comment:
It would be good to identify the executable. Not sure atm which parameter to use for that. launcherName or productId. WDYT?

Reply:
Thinking about it more, I think, the productId should be mandatory and a unique identifier of the server.

Reply:
I think `launcherName` is more appropriate for the executable. For example, launching the server from Eclipse gives a `launcherName` = `Eclipse`, while launching it from the packaged applications gives `launcherName` = `Tracecompass-serverc`

Reply:
`productId` is now mandatory 👍